tests/unittests/RecommendationSystemTest.cpp

```diff
@@ -135,6 +140,39 @@ llvm::cl::opt<bool> dumpBinaryResults(
     llvm::cl::init(false), llvm::cl::cat(recSysTestCat));
 } // namespace
 
+class TestDeferredWeightLoader : public DeferredWeightLoader {
+public:
+  Error loadNextWeight() override {
+    position_++;
+    return Error::success();
+  }
+  Error setSrc(void *loaderObject) override { return Error::success(); }
+  void addWeight(Tensor *weight) { weights_.push_back(weight); }
+  void addName(std::string name) { names_.push_back(name); }
+  void setTypeInfo(std::map<std::string, Type> info) override {}
+
+  std::string getName() override {
+    for (auto na : names_) {
+    }
```

Comment:
What's this for, rm it?

Reply:
Not sure, some leftovers - removed + made another small fix (changed the names given to the loader to be unique).